Runtime/Setup/e2esetup.js

```diff
@@ -41,6 +42,7 @@ function usage() {
   console.log("      --tier:              The tier level for the app i.e Free, basic or standard");
   console.log("      --pingEndpoint:      Endpoint to ping (GET) after app is created (e.g. 'status' or 'tables/movies')");
   console.log("      --pingDelay:         Time (in milliseconds) to wait before pinging the pingEndpoint (default: " + defaultPingDelay + ")");
+  console.log("      --pushSetupDelay:    Time (in milliseconds) to wait before pinging the pingEndpoint (default: " + defaultPushSetUpDelay + ")");
```

Comment:
Incorrect description of setting.  Also, you may want to indicate as well that the problem is due to notification hub activation